src/peep/staff.c

```diff
@@ -253,7 +256,7 @@ void game_command_hire_new_staff_member(int* eax, int* ebx, int* ecx, int* edx,
 		newPeep->pathfind_goal.z = 0xFF;
 		newPeep->pathfind_goal.direction = 0xFF;
 
-		uint8 colour = gStaffColours[staff_type > 2 ? 2 : staff_type];
+		uint8 colour = gStaffColours[clamp(0, staff_type, STAFF_TYPE_SECURITY)];
```

Comment:
Mostly I tried getting rid of *magic* constants in the code. `STAFF_TYPE_SECURITY` is `2`. What the original code basically does is set color to `[2]` if `staff_type` is `STAFF_TYPE_ENTERTAINER` for some reason. Is essentially clamping the value to `[0, 2]`, so I tried to make that more explicit.